sample_datasets/snips-to-rasa.py

```diff
@@ -0,0 +1,67 @@
+import json
+
+inputFiles = [
+    '2017-06-custom-intent-engines/GetWeather/train_GetWeather_full.json',
+    '2017-06-custom-intent-engines/AddToPlaylist/train_AddToPlaylist_full.json',
+    '2017-06-custom-intent-engines/BookRestaurant/train_BookRestaurant_full.json',
+    '2017-06-custom-intent-engines/GetWeather/train_GetWeather_full.json',
+    '2017-06-custom-intent-engines/PlayMusic/train_PlayMusic_full.json',
+    '2017-06-custom-intent-engines/RateBook/train_RateBook_full.json',
+    '2017-06-custom-intent-engines/SearchCreativeWork/train_SearchCreativeWork_full.json',
+    '2017-06-custom-intent-engines/SearchScreeningEvent/train_SearchScreeningEvent_full.json',
+]
+
+for inputFile in inputFiles:
+    print('Now converting: ' + inputFile)
+
+    with open(inputFile, 'r', encoding='utf8') as f:
+        data = json.loads(f.read())
+
+    for intent in data:
+        examples = data[intent]
+        outputExamples = []
+
+        for example in examples:
+            example = example['data']
+
+            text = ''
+            textLength = 0
+            entities = []
+            exampleLength = len(example) - 1
+
+            for index, phrase in enumerate(example):
+                # print(index, exampleLength)
+
+                if index != exampleLength:  # or (index == exampleLength and phrase['text'] != ' '):

```

Comment:
whats the comment?

Reply:
Some datasets had strange spaces at the end of some sentences, so I made a _really_ dodgy workaround